src/app/watch/[id]/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import Player from "@/components/Watch/Player/Player";
+import { useState, useEffect } from "react";
+
+export default function Watch(props: { params: { id: string } }) {
```

Comment:
```suggestion
type WatchParams = {
  id: string;
}

export default function Watch(props: WatchParams) {
```
여기도 props 따로 빼주면 좋을듯요
참고로 props를 위한 타입을 정의할 때는 `Interface`보다 `type`을 이용하는 경우가 많습니다. 둘 다 객체 타입을 정의하는 방식이지만 interface가 type보다 갖고 있는 기능이 많아서 그러한데요, 자세한 부분은 [공식문서](https://www.typescriptlang.org/docs/handbook/2/objects.html#interfaces-vs-intersections) 참고